recipes/seekr2_openmm_plugin/build.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+CMAKE_FLAGS="-DCMAKE_INSTALL_PREFIX=$PREFIX -DCMAKE_BUILD_TYPE=Release -DSEEKR2_BUILD_OPENCL_LIB=OFF -DOPENMM_DIR=$PREFIX"
+
+if [[ "$target_platform" == linux* ]]; then
+    # CFLAGS
+    # JRG: Had to add -ldl to prevent linking errors (dlopen, etc)
+    MINIMAL_CFLAGS+=" -O3 -ldl"
+    CFLAGS+=" $MINIMAL_CFLAGS"
+    CXXFLAGS+=" $MINIMAL_CFLAGS"
+    LDFLAGS+=" $LDPATHFLAGS"
+
+    # Use GCC
+    CMAKE_FLAGS+=" -DCMAKE_C_COMPILER=$CC -DCMAKE_CXX_COMPILER=$CXX"
+
+    # CUDA_HOME is defined by nvcc metapackage
+    CMAKE_FLAGS+=" -DCUDA_TOOLKIT_ROOT_DIR=${CUDA_HOME}"
+    # From: https://github.com/floydhub/dl-docker/issues/59
+    CMAKE_FLAGS+=" -DCMAKE_LIBRARY_PATH=${CUDA_HOME}/lib64/stubs"
+    CMAKE_FLAGS+=" -DCUDA_CUDA_LIBRARY=${CUDA_HOME}/lib64/stubs/libcuda.so"
+    
+    # Cuda tests won't build. Disable all tests for now
+    CMAKE_FLAGS+=" -DSEEKR2_CUDA_BUILD_TESTS=OFF"
+    CMAKE_FLAGS+=" -DSEEKR2_REFERENCE_BUILD_TESTS=OFF"
+    CMAKE_FLAGS+=" -DSEEKR2_BUILD_SERIALIZATION_TESTS=OFF"
+fi
+
+mkdir build
+cd build
+cmake ${CMAKE_FLAGS} ${SRC_DIR}/seekr2plugin
```

Comment:
What is the reason for this change, @xhochy? I think it would be a bad idea to change this word, because it is the default variable used in the compilation of OpenMM and also this package.

Reply:
Oh I had not seen your comment below yet...

Reply:
We can try it and see whether it works...